tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Canonical Ltd.
+# Licensed under the GPLv3, see LICENCE file for details.
+
+import pytest
+
+
+def pytest_addoption(parser: pytest.Parser):
+    # --openstack-rc points to an openstack credential file in the "rc" file style
+    # Here's an example of that file
```

Comment:
Do you think it would improve maintainability if the comment just points to the `openstack.rc` file that is also in this PR?

Reply:
The openstack.rc file will be deleted. We should point to the file generated by the local devstack installation, and that will vary between local machines.